inference_onnx.py

```diff
@@ -0,0 +1,243 @@
+"""
+python3 ./inference_onnx.py \
+    --checkpoint "rvm_mobilenetv3_fp32.onnx" \
+    --device cpu \
+    --dtype fp32 \
+    --input-source "input.mp4" \
+    --downsample-ratio 0.25 \
+    --output-type video \
+    --output-composition "composition.mp4" \
+    --output-alpha "alpha.mp4" \
+    --output-foreground "foreground.mp4" \
+    --output-video-mbps 4 \
+    --seq-chunk 1
+"""
+
+import torch
+import os
+from torch.utils.data import DataLoader
+from torchvision import transforms
+from typing import Optional, Tuple, Union, Type
+from tqdm.auto import tqdm
+import onnxruntime as ort
+import numpy as np
+
+from inference_utils import VideoReader, VideoWriter, ImageSequenceReader, ImageSequenceWriter
+
+
+def convert_video(session: ort.InferenceSession,
+                  input_source: str,
+                  input_resize: Optional[Tuple[int, int]] = None,
+                  downsample_ratio: Optional[float] = None,
+                  output_type: str = 'video',
+                  output_composition: Optional[str] = None,
+                  output_alpha: Optional[str] = None,
+                  output_foreground: Optional[str] = None,
+                  output_video_mbps: Optional[float] = None,
+                  seq_chunk: int = 1,
```

Comment:
This is no longer applicable, because only PyTorch model supports [B, T, C, H, W] format. ONNX only supports [B, C, H, W]. I don't think you are currently handling seq_chunk > 1 case, so there is probably a bug. This argument can be removed.